src/VisualStudio/Core/Test/ProjectSystemShim/VisualBasicHelpers/MockCompilerHost.vb

```diff
@@ -20,6 +20,12 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim.Vi
             End Get
         End Property
 
+        Public Shared ReadOnly Property NoSdkCompilerHost As MockCompilerHost
```

Comment:
Nit: Could just add an `int` property called SdkPathReturn and use that as the return of `GetSdkPath`.  That way users of the Mock could fully control the return of the method call.  Allows for them to create scenarios where it returns an error but a valid non-null path for example.